src/aat/resources/features/F-048/F-048.feature

```diff
@@ -0,0 +1,56 @@
+@F-048
+Feature: Get the pagination metadata for a case data search for Case Worker
```

Comment:
Scenarios doesn't provide specs about test data, and read too generic.

Reply:
Added the specs for all test cases. Will uncomment the request specs once DSL is released.